internal/cli/main.go

```diff
@@ -180,7 +183,12 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n", createUsage, gotoU
 			log.fatal("error: -ext flag must be specified")
 		}
 
-		if err := createCmd(*dirPtr, startTime, *formatPtr, name, *extPtr, seq, seqDigits, true); err != nil {
+		timezone, err := time.LoadLocation(*timezoneName)
```

Comment:
Let's avoid loading TZ data if the named tz is the default. That way, missing tz data won't break the CLI if a tz isn't specified.

Reply:
@dhui on another thought, I cannot see how this will break, since the default location will be UTC even if you do not specify the timezone. Essentialy we will do `time.LoadLocation("UTC")`

Reply:
The issue is when the tz database is missing, not when the `-tz` option is not specified or invalid/incorrect.

From the `time.LoadLocation()` [docs](https://golang.org/pkg/time/#LoadLocation):
> The time zone database needed by LoadLocation may not be present on all systems, especially non-Unix systems. LoadLocation looks in the directory or uncompressed zip file named by the ZONEINFO environment variable, if any, then looks in known installation locations on Unix systems, and finally looks in $GOROOT/lib/time/zoneinfo.zip.

Reply:
Oh, I completely misinterpreted what you were referring to. I think we should be fine still since `zoneinfo` does a look upon an IANA file only if the location name is neither `'UTC'` nor `'Local'` https://github.com/golang/go/blob/master/src/time/zoneinfo.go#L630-L635

Reply:
Thanks for digging deeper and discovering that the tz db isn't loaded if the location `UTC` or `Local` are specified! Looks like this behavior has been around since [at least Go 1.4](https://github.com/golang/go/blob/go1.4/src/time/zoneinfo.go#L273) and seems unlikely to change in the future, so I think the current approach is fine. The fix is easy enough to make should this break in the future.